doc/source/user_documentation.rst

```diff
@@ -535,6 +531,105 @@ Here is an example of connectivity rules:
 	    'delay': [5, 10],                             # different delays for each of 3 synapses per synMech 
 	    'loc': [[0.1, 0.5, 0.7], [0.3, 0.4, 0.5]]}    # different locations for each of the 6 synapses
 
+.. _multisynapse_conn:
+
+**Connectivity Rules with Multiple Synapses per Connection**
+
+By default, there is only one synapse created per connection. However, multiple synapses can be created targeting each post-synaptic cell. This can be achieved by setting ``synsPerConn`` to a value greater than 1, or by defining ``synMech`` as a list, or both.
+
+* Setting ``synsPerConn`` >= 1 with ``synMech`` as a single value
+	If ``synsPerConn`` is > 1, then for each connection, this number of synapses will be created. The weights and delays can be specified as lists, or a single value can be used for all. ``loc`` should be a list, a single value is not accepted. If ``loc`` is omitted, a uniform distribution is used (i.e., synapses are placed at equal distances along the section).
+
+	.. code-block:: python
+
+		netParams.connParams['PYR->PYR'] = {
```

Comment:
might be useful to include: synMech: ['AMPA'] and sec: ['soma'] ? or you think more confusing?

Reply:
also, maybe similar example code blocks for the more complex rules in this subsection might be useful?